content/blog/entries/cc-vocabulary-docs-intro/contents.lr

```diff
@@ -0,0 +1,62 @@
+title: Vocabulary Site & Usage Guide Introduction (GSoD'20)
+---
+categories:
+cc-vocabulary
+gsod
+gsod-2020
+---
+author: nimishbongale
+---
+series: gsod-2020-vocabulary-usage-guide
+---
+pub_date: 2020-10-02
+---
+body:
+Hey there! I'm Nimish Bongale, a Technical Writer & Software Developer based out of Bangalore, India. My other hobbies include playing chess and the guitar. I look forward to build the CC Vocabulary site and usage guides as a part of GSoD'20.
+
+## But what is GSoD?
+
+GSoD, or Google Season of Docs, is a program that stresses on the importance of the documentation aspect of Open Source projects. It invites technical writers from across the world to submit proposals based on projects floated in by the participating Open Source Organisations. The selected technical writers then work with the their respective organisations and look to complete their work by the end of their internship period. More information about the same can be found [here](https://developers.google.com/season-of-docs).
+
+Let's talk a bit about my project, shall we?
+
+## Vocabulary Site & Usage Guide
+
+### Introduction
+[CC Vocabulary](https://github.com/creativecommons/vocabulary) is a cohesive design system & Vue component library to unify the web-facing Creative Commons. It's currently comprised of 3 packages, namely Vocabulary, Vue-Vocabulary & Fonts. My contribution to this project would majorly involve building the landing site for CC Vocabulary, and refactor the documentation wherever necessary.
+
+### What drives me
+Documentation is one of the primary reasons which determines how successful a certain open source library will be. The major question that developers think of while choosing a suitable tech stack to build their applications is:
+
+- Is the library _well documented_? 
+- Is it _well maintained_? 
+- Does it have some _considerable usage and error support_?
+
+These are exactly the questions I should be asking myself while going about this project idea.
+
+As aforementioned, there is an immanent need to have a concise and consolidated documentation. The lack of documentation hurts the future perspectives of open source applications, and is by far, an essential and non-negligible component. Linking to these documentations should be an appealing home page, which captures the interest of the people in an instant. The documentation should be well organised, thereby enabling a seamless flow through it.
```

Comment:
*imminent 😄

Reply:
I think I mean _immanent_ here if I'm not mistaken 🤔

imminent = obvious & approaching
immanent = inherent / innate

> There is an inherent need...

This is what I wanted it to mean, but turns out even if I replace `immanent` with `imminent` the sentence would still make sense, but in a very different way 😄